lighthouse-core/audits/byte-efficiency/render-blocking-resources.js

```diff
@@ -25,6 +26,19 @@ const NetworkRequest = require('../../lib/network-request');
 // to possibly be non-blocking (and they have minimal impact anyway).
 const MINIMUM_WASTED_MS = 50;
 
+const UI_STRINGS = {
+  title: 'Eliminate render-blocking resources',
+  description: 'Resources are blocking the first paint of your page. Consider ' +
+  'delivering critical JS/CSS inline and deferring all non-critical ' +
+  'JS/styles. [Learn more](https://developers.google.com/web/tools/lighthouse/audits/blocking-resources).',
+  displayValue: `{itemCount, plural,
+    one {1 resource}
+    other {# resources}
+    } delayed first paint by {timeInMs, number, milliseconds} ms`
+};
+
+const i18n = i18nUtils.createStringFormatter(__filename, UI_STRINGS);
```

Comment:
this is a formatting function that has context bundled in explicitly via `__filename` and `UI_STRINGS`